code_samples/background_tasks/src/Dispatcher/SomeClassThatSchedulesExecutionInTheBackground.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace App\Dispatcher;
+
+use Ibexa\Messenger\Stamp\DeduplicateStamp;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class SomeClassThatSchedulesExecutionInTheBackground
+{
+    private MessageBusInterface $bus;
+
+    public function __construct(MessageBusInterface $bus)
+    {
+        $this->bus = $bus;
+    }
```

Comment:
For 5.0, we can write
```suggestion
    public function __construct(private MessageBusInterface $bus)
    {
    }
```